mscore/plugin/pluginManager.cpp

```diff
@@ -188,7 +185,10 @@ void PluginManager::updatePluginList(bool forceRefresh)
       QList<QString> pluginPathList;
       pluginPathList.append(dataPath + "/plugins");
       pluginPathList.append(mscoreGlobalShare + "plugins");
-      pluginPathList.append(preferences.getString(PREF_APP_PATHS_MYPLUGINS));
+      QStringList ps = preferences.getString(PREF_APP_PATHS_MYPLUGINS).split(";");
+      for(int i=0; i < ps.size(); i++)
```

Comment:
space between `for` and `(` please

Reply:
Why not use
```cpp
      for (ps : references.getString(PREF_APP_PATHS_MYPLUGINS).split(";")))
            pluginPathList.append(ps);
```

Reply:
I do wonder however why there would ever be a list of multiple directories there, as far as I can tell there's only one entry possible in Edit > Preferences > General > Directories > Plugins (multiple directories are only possible for SoundFonts)

Reply:
So the "list" is one or none, isn't it?

Reply:
Although I like the idea of multiple paths, that's require some more code changes though

Reply:
Not sure though whether that'd cause issues with 3.7 sharing its settings with 3.6